release/go-mod-bootstrap.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 name: 'go-mod-bootstrap'
-version: '2.1.0'
+version: '2.1.1'
```

Comment:
Bootstrap need to be release separately once it has been updated with the released go-mod versions for the other two.